app/assets/javascripts/components/AccountMenu.tsx

```diff
@@ -0,0 +1,984 @@
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordConfirmation, setPasswordConfirmation] = useState<string | undefined>(undefined);
+  const [status, setStatus] = useState<string | undefined>(undefined);
+  const [syncError, setSyncError] = useState<string | undefined>(undefined);
+
+  const [isEphemeral, setIsEphemeral] = useState(false);
+  const [isStrictSignIn, setIsStrictSignIn] = useState(false);
+
+  const [passcode, setPasscode] = useState<string | undefined>(undefined);
+  const [passcodeConfirmation, setPasscodeConfirmation] = useState<string | undefined>(undefined);
+
+  const [encryptionStatusString, setEncryptionStatusString] = useState<string | undefined>(undefined);
+  const [isEncryptionEnabled, setIsEncryptionEnabled] = useState(false);
+  const [shouldMergeLocal, setShouldMergeLocal] = useState(true);
+
+  const [server, setServer] = useState<string | undefined>(undefined);
+  const [url, setUrl] = useState<string | undefined>(undefined);
+  const [showPasscodeForm, setShowPasscodeForm] = useState(false);
+  const [selectedAutoLockInterval, setSelectedAutoLockInterval] = useState<unknown>(null);
+  const [isImportDataLoading, setIsImportDataLoading] = useState(false);
+  const [isErrorReportingEnabled, setIsErrorReportingEnabled] = useState(false);
+  const [appVersion, setAppVersion] = useState('');
+  const [hasPasscode, setHasPasscode] = useState(application.hasPasscode());
+  const [isBackupEncrypted, setIsBackupEncrypted] = useState(isEncryptionEnabled);
+  const [isSyncInProgress, setIsSyncInProgress] = useState(false);
+  const [protectionsDisabledUntil, setProtectionsDisabledUntil] = useState(getProtectionsDisabledUntil());
+
+  const reloadAutoLockInterval = useCallback(async () => {
+    const interval = await application.getAutolockService().getAutoLockInterval();
+    setSelectedAutoLockInterval(interval);
+  }, [application]);
+
+  const user = application.getUser();
+  const errorReportingIdValue = errorReportingId();
+  const canAddPasscode = !application.isEphemeralSession();
+  const keyStorageInfo = StringUtils.keyStorageInfo(application);
+  const passcodeAutoLockOptions = application.getAutolockService().getAutoLockIntervalOptions();
+  const showBetaWarning = appState.showBetaWarning;
+
+  const focusWithTimeout = (inputElementRef: RefObject<HTMLInputElement>) => {
+    // In case the ref element is not yet available at this moment,
+    // we call `focus()` after timeout.
+    setTimeout(() => {
+      inputElementRef.current && inputElementRef.current.focus();
+    }, 0);
+  };
+
+  const handleSignInClick = () => {
+    setShowLogin(true);
+    focusWithTimeout(emailInputRef);
+  };
+
+  const handleRegisterClick = () => {
+    setShowRegister(true);
+    focusWithTimeout(emailInputRef);
+  };
+
+  const blurAuthFields = () => {
+    emailInputRef.current.blur();
+    passwordInputRef.current.blur();
```

Comment:
We need to blur the password confirmation input here too

Reply:
Added